src/foam/nanos/auth/UserPropertyAvailabilityService.js

```diff
@@ -40,13 +40,20 @@ foam.CLASS({
           throw new AuthorizationException();
         }
 
-        ArraySink select = (ArraySink) userDAO.inX(x).where(MLang.EQ(targetProperty.equals("userName") ? User.USER_NAME : User.EMAIL, value)).select(new ArraySink());
-        
+        ArraySink select = (ArraySink) userDAO.inX(x).where(
+          MLang.AND(
+            MLang.EQ(targetProperty.equals("userName") ? User.USER_NAME : User.EMAIL, value),
```

Comment:
If you add ''static foam.mlang.MLang.*' to your javaImports: then you don't need to prefix all mlangs with 'MLang.'.

Reply:
If you get class info of the User model then you can find the axiom by name instead of conditional operator.

Reply:
Thank you! It's good to know.